test/execute.js

```diff
@@ -529,13 +529,60 @@ describe('execute', () => {
         headers: {
           head: 'justTheHead',
         },
-        body: {
+        body: JSON.stringify({
           json: 'rulez'
-        }
+        })
       })
     })
   })
 
+  it('should pass through the parameters object as the value if parameter is in the body', function () {
```

Comment:
Yeah, that is a little confusing.  I mean that the whole `parameters` object will be used as the parameter value instead of `parameters[parameter.name]`. I'll clarify the wording.  To your second question, the difference in this test is the parameters object in the buildRequest call below.  

In the other tests it is:
`parameters: {
          ...,
          body: {
        name: 'johny',
        id: '123'
      },
        }`

in this test it is:
`parameters: {
        name: 'johny',
        id: '123'
      }`

This allows you to call operations like `client.apis.default.operation({name: 'johny', id: '123'})` if there is a body parameter.